laravel-app/app/Http/Requests/UpdateProfileRequest.php

```diff
@@ -8,9 +8,9 @@
 class UpdateProfileRequest extends FormRequest
 {
     /**
-     * Get the validation rules that apply to the request.
      *
-     * @return array<string>
+     *
+     * @return array<string, mixed>
      */
     public function rules(): array
```

Comment:
Since they are using Laravel 10. Laravel already removed doc block and used return type declaration.

Docs:
https://laravel.com/docs/10.x/releases#types
https://www.php.net/manual/en/language.types.declarations.php

Demo:
https://www.reddit.com/r/laravel/comments/11328e0/quick_demo_of_the_laravel_10x_shift/